subprojects/packagefiles/iir1/iir/meson.build

```diff
@@ -0,0 +1,41 @@
+sources = files(
+  'Biquad.cpp',
+  'Butterworth.cpp',
+  'Cascade.cpp',
+  'ChebyshevI.cpp',
+  'ChebyshevII.cpp',
+  'Custom.cpp',
+  'PoleFilter.cpp',
+  'RBJ.cpp')
+
+cpp_args = []
+cxx = meson.get_compiler('cpp')
+host_cpu_family = host_machine.cpu_family()
+
+# Maybe disable exceptions
+if not get_option('exceptions')
```

Comment:
Lovely! Upstream doesn't have an option and simply documents using this in README...

Reply:
👍